test/extensions/filters/network/ext_authz/ext_authz_fuzz.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+package envoy.extensions.filters.network.ext_authz;
+
+import "envoy/extensions/filters/network/ext_authz/v3/ext_authz.proto";
+import "google/protobuf/empty.proto";
+import "validate/validate.proto";
+
+message Result {
+  oneof result_selector {
+    option (validate.required) = true;
+    // Authorization check status
+    google.protobuf.Empty check_status_error = 1;
+    google.protobuf.Empty check_status_denied = 2;
+    google.protobuf.Empty check_status_ok = 3;
+  }
+}
+
+message OnData {
+  bytes data = 1 [(validate.rules).bytes = {min_len: 1}];
```

Comment:
is giving empty data a case that cannot occur? (does the filter manager check ahead of time?)

Reply:
> is giving empty data a case that cannot occur? (does the filter manager check ahead of time?)

Empty data can occur. Fixed. Thanks!